src/boot/efi/boot.c

```diff
@@ -524,14 +527,9 @@ static BOOLEAN menu_run(
         Print(L" ");
 
         err = console_set_mode(config->console_mode, config->console_mode_change);
-        uefi_call_wrapper(ST->ConOut->ClearScreen, 1, ST->ConOut);
-        if (EFI_ERROR(err))
-                PrintError(L"Error switching console mode to %lu: %r.\n", config->console_mode, err);
-
-        err = uefi_call_wrapper(ST->ConOut->QueryMode, 4, ST->ConOut, ST->ConOut->Mode->Mode, &x_max, &y_max);
         if (EFI_ERROR(err)) {
-                x_max = 80;
-                y_max = 25;
+                uefi_call_wrapper(ST->ConOut->ClearScreen, 1, ST->ConOut);
+                PrintError(L"Error switching console mode to %lu: %r.\n", config->console_mode, err);
```

Comment:
i figure part of this hunk should actually be in a patch earlier, where yu added this.